config/validator/service/prepare_beacon_proposer_config.go

```diff
@@ -0,0 +1,10 @@
+package validator_service_config
```

Comment:
Add some comments to these structs to explain what they are for. Maybe name the file to fee_recipients_config.go. We want to make it clear this is about fee recipients and not about something else proposer-related